File: Iniciante/1131 - Grenais/Python/1131.py
def main():
    NumberOfGames = 0
    interVictories = 0
    gremioVictories = 0
    draws = 0

    while True:
        interGoals, gremioGoals = map(int, input().split())

        NumberOfGames += 1

        if interGoals == gremioGoals:
            draws += 1
        elif interGoals > gremioGoals:
            interVictories += 1
        else:
            gremioVictories += 1

        while True:
            print("Novo grenal (1-sim 2-nao)")
            code = int(input())
            if code == 1 or code == 2:
                break

        if code == 2:
            break

    print(f"{NumberOfGames} grenais")
    print(f"Inter:{interVictories}")
    print(f"Gremio:{gremioVictories}")
    print(f"Empates:{draws}")

    if interVictories == gremioVictories:
        print("Nao houve vencedor")
    elif interVictories > gremioVictories:
        print("Inter venceu mais")
    else:
        print("Gremio venceu mais")


if __name__ == "__main__":
    main()

File: Iniciante/1131 - Grenais/JavaScript/1131.js
function main() {
    const input = require('fs').readFileSync(0, 'utf-8').trim().split(/\s+/).map(Number);

    let i = 0;

    let NumberOfGames = 0;
    let interVictories = 0;
    let gremioVictories = 0;
    let draws = 0;

    while (true) {
        let interGoals = input[i++];
        let gremioGoals = input[i++];

        NumberOfGames++;

        if (interGoals === gremioGoals) {
            draws++;
        } else if (interGoals > gremioGoals) {
            interVictories++;
        } else {
            gremioVictories++;
        }

        let code;

        while (true) {
            console.log("Novo grenal (1-sim 2-nao)");
            code = input[i++];
            if (code === 1 || code === 2) break;
        }

        if (code === 2) break;
    }

    console.log(`${NumberOfGames} grenais`);
    console.log(`Inter:${interVictories}`);
    console.log(`Gremio:${gremioVictories}`);
    console.log(`Empates:${draws}`);

    if (interVictories === gremioVictories) {
        console.log("Nao houve vencedor");
    } else if (interVictories > gremioVictories) {
        console.log("Inter venceu mais");
    } else {
        console.log("Gremio venceu mais");
    }
}

main();